### PR TITLE
Dark Vision and You: How I learned to hate Elves even more

### PR DESCRIPTION
**Pull request by SELFHELL**
Elves and their halfblooded kin have regained the blessing of Darksight, no longer will mild inconveniences like losing a torch bother the superior race(s)

Dwarves now heal rather quickly from alcohol consumption

### CODE REVIEW

**Comment by SpaceSlug2** [comment]
Give this to Dark Elf aswell.

**Comment by SELFHELL** [comment]
I... did?

**Comment by SpaceSlug2** [comment]
or did you

**Comment by SpaceSlug2** [comment]
curious

**Comment by SELFHELL** [comment]
should be good for merge

**Comment by TopHatPenguin** [comment]
the elf population boom when this gets merged

File: code/modules/mob/living/carbon/human/species_types/roguetown/other/halfelf.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	string_to_replace_skintone_on_charpref = "Identifies As"
 	default_color = "FFFFFF"
 	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS,STUBBLE,OLDGREY)
-	inherent_traits = list(TRAIT_NOMOBSWAP)
+	inherent_traits = list(TRAIT_NOMOBSWAP,TRAIT_NIGHT_VISION)
```
Comment (ChungusGamer666) [inline code/modules/mob/living/carbon/human/species_types/roguetown/other/halfelf.dm:21]: Helves should have no night vision, or shittier night vision

Comment (SELFHELL) [reply code/modules/mob/living/carbon/human/species_types/roguetown/other/halfelf.dm]: Alright, Replaced with the New "TRAIT_LESSER_NIGHT_VISION" with a decent drop in the alpha

**Comment by Osmos123** [comment]
I feel like there should be more of an counterbalance for the addition of night vision for elves. The elves racial attributes leaned heavily and benefitted being ranged units such as rogues and rangers. The night vision would be a big buff. I wonder if you can add in any penalty for the elves.

Otherwise, it feels like it is needs to be more acceptable in the games culture to brutally kill elves that are equipped with ranged weapons.

**Comment by SELFHELL** [comment]
Of the moment, the plan is to add racial traits for everything (except) humans that's a direct benefit; already added dwarvern alcohol healing in this PR.

**Comment by Osmos123** [comment]
Fair enough, in that case with the racial traits for all except humans I think it is good to merge. I believe most of the high level/ranking positions are human only so it balances out or could be made more stringent to human only.

**Comment by ChungusGamer666** [comment]
slop lore is so good

**Pull request merged**